src/components/home/index.tsx

```diff
@@ -1,9 +1,11 @@
 import React from 'react';
-import Hero from './hero';
+import Hero from './hero/hero';
+import Cards from './cards/cards';
 
 const Home = () => (
     <>
```

Comment:
Here it makes sense, since we don't want to insert a standalone div to wrap up Hero and Cards